src/server/app/server.ts

```diff
@@ -54,20 +53,26 @@ app.get('/title/:id', function (req, res) {
     });
 });
 
-app.get('/board/:id', function (req, res) {
-    var ID = req.query.id;
-    ID = "ID" + ID;
+app.get('/board', function (req, res) {
+    //var idArray = [1,2];
+    var idArray = req.query;
     fs.readFile(dataFile, 'utf8', function (err, data) {
         var daten = JSON.parse(data);
         var board: IBoard;
+        var dataArray = [];
```

Comment:
Consider renaming this variable. The type already says its an array. You could call it ```ids ``` or ``` requestedIds```.